integrations/acquisition/covidcast/test_delete_batch.py

```diff
@@ -0,0 +1,122 @@
+"""Integration tests for covidcast's batch deletions."""
+
+# standard library
+from collections import namedtuple
+import unittest
+from os import path
+
+# third party
+import mysql.connector
+
+# first party
+from delphi_utils import Nans
+from delphi.epidata.client.delphi_epidata import Epidata
+import delphi.operations.secrets as secrets
+
+# py3tester coverage target (equivalent to `import *`)
+__test_target__ = 'delphi.epidata.acquisition.covidcast.database'
+
+Example = namedtuple("example", "given expected")
+
+class DeleteBatch(unittest.TestCase):
+    """Tests batch deletions"""
+
+
+    def setUp(self):
+        """Perform per-test setup."""
+
+        # connect to the `epidata` database and clear the `covidcast` table
+        cnx = mysql.connector.connect(
+            user='user',
+            password='pass',
+            host='delphi_database_epidata',
+            database='epidata')
+        cur = cnx.cursor()
+        cur.execute('truncate table covidcast')
+        cnx.commit()
+        cur.close()
+
+        # make connection and cursor available to test cases
+        self.cnx = cnx
+        self.cur = cnx.cursor()
+
+        # use the local instance of the epidata database
+        secrets.db.host = 'delphi_database_epidata'
+        secrets.db.epi = ('user', 'pass')
+
+        # use the local instance of the Epidata API
+        Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
+
+        # will use secrets as set above
+        from delphi.epidata.acquisition.covidcast.database import Database
```

Comment:
move to top of file, unless there's some reason it needs to happen here

Reply:
If we do it earlier, Database uses the default Epidata.BASE_URL, and not the one set on line 48